deepdoc/server/deepdoc_server.py

```
@@ -47,43 +57,99 @@ def parse_args():
+    data = await file.read()
+    result = tsr_adapter(data)
+    return {"bboxes": result}
+
+
+def main():
+    args = parse_args()
+    logging.getLogger().setLevel(getattr(logging, args.log_level.upper(), "INFO"))
 
     logger.info("Starting server on port %d...", args.port)
-    server.run(port=args.port)
+    uvicorn.run(app, host="0.0.0.0", port=args.port, timeout_keep_alive=args.timeout)
```

Ruff (0.16.1)

[error] Possible binding to all interfaces

(S104)